File: edut3c/back-end/models/level.py
from db.database import db

class Level(db.Model):
    
    __tablename__ = "level"
 
    id = db.Column(
        db.Integer,
        primary_key=True,
    )

    name = db.Column(
        db.String(100),
        nullable=False,
    )

    xp_reward = db.Column(
        db.Integer,
        nullable=False,
    )

    bits_reward = db.Column(
        db.Integer,
        nullable=False,
    )
    
def to_dict(self):
    return {
        "id": self.id,
        "name": self.name,
        "xp_reward": self.xp_reward,
        "bits_reward": self.bits_reward
    }

File: edut3c/lib/models/level.dart
class Level {

  final int id;
  final String name;
  final int xpReward;
  final int bitsReward;

  Level({
    required this.id,
    required this.name,
    required this.xpReward,
    required this.bitsReward,
  });

  factory Level.fromJson(Map<String, dynamic> json) {
    return Level(
      id: json["id"],
      name: json["name"],
      xpReward: json["xp_reward"],
      bitsReward: json["bits_reward"],
    );
  }
}